src/integrationTests/runWithoutTor.ts

```diff
@@ -0,0 +1,32 @@
+import debug from 'debug';
+import communityTestCases from './community';
+import { prepareStore } from './utils';
+const log = Object.assign(debug('tests'), {
+  error: debug('tests:err'),
+});
+
+function testCaseReducer(
```

Comment:
I don't really like this copy pasting because we would have to copy it for every test we write. 
We can leave it for now but we have to write a more flexible solution that runs all tests from one place.